### ISSUE TITLE
Restarting taker crashes

### ISSUE THREAD

**Issue by bonomat** [get10101/itchysats#391]
I stopped the taker and restarted it and got the following error: 

```bash

     Running `target/debug/taker --data-dir=/Users/bonomat/src/github/comit/hermes/data/taker --maker '116.240.228.91:9999' testnet`
2021-10-20 11:41:25  INFO Initialized logger
2021-10-20 11:41:25  INFO Running version: preview-150-gd2b1e11
2021-10-20 11:41:29  INFO Connected to BitMex realtime API
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: error occurred while decoding column 0: invalid utf-8 sequence of 1 bytes from index 2

Caused by:
    invalid utf-8 sequence of 1 bytes from index 2', daemon/src/taker.rs:199:22
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

which is this line: https://github.com/comit-network/hermes/blob/7c6197f7e23ce694a98604c707135a4a1eba6b74/daemon/src/taker.rs#L197-L199

Note: I cannot start the taker anymore without deleting the database.

**Comment by da-kami** [comment]
That might have been due to the rebase you did in the morning. The rebase changed the `Uuid` from bytes to `hyphenated` `strings`. 

My guess is: 
1. You were on branch `debug-statements` which was not rebased on `master` where @thomaseizinger's https://github.com/comit-network/hermes/pull/383 merged.
You started the taker, connected to a maker and received an order. That order was saved in the database. You hit buy and a cfd was created (but then we stopped).

2. You rebased on `master`, which changed the way `OrderId` is stored in the db. I think `invalid utf-8 sequence of 1 bytes from index 2` refers to the `Uuid` index. 

This is of course just a guess. We can keep it open for now, but I think we should not see this anymore.
